### PR TITLE
Timeseries assert --> error fix

### PR DESCRIPTION
**Pull request by nicolejkeeney**
This PR addresses an [issue raised by Eric](https://github.com/cal-adapt/climakitae/issues/108) to replace the assert statements with error messages. I also improved the error messages. 

**To test:** Try running the timeseries_examples notebook and loading the wrong data. The following line should raise an error with an appropriate error message if you feed it "bad" data (i.e. data that can't be used by the timeseries tool) 
```
timeseries = tst.Timeseries(my_data)
```

### CODE REVIEW

**Review by vicford — CHANGES_REQUESTED** [review]
Not quite sure what the desired behavior is if multiple of these errors are relevant. 

I tested with data that was not area averaged, and included SSP3-7.0 but I did not select "append historical". I got one of (presumably) two errors:
`ValueError: Please pass a timeseries (area average).`

**Review by vicford — APPROVED** [review]
Better! 
Screen Shot 2022-11-10 at 12 50 44 PM

File: climakitae/timeseriestools.py [hunk 1]
```diff
@@ -253,16 +253,23 @@ class Timeseries:
...
-        ), "Please pass a timeseries (area average or individual station)."
-        assert (
-            True in ["Historical + " in v for v in data.scenario.values]
-        ), "Please append the historical period in your data retrieval."
-
+        
+        # Confirm that the user has input a valid dataset
+        not_xr_da = type(data) != xr.core.dataarray.DataArray # Data is NOT in the form of xr.DataArray 
+        not_area_averaged = "lat" in data.coords # Data is NOT area averaged 
+        not_append_historical = any(["Historical + " in v for v in data.scenario.values]) == False # Append historical = False
```
Comment (elehmer) [inline climakitae/timeseriestools.py:260]: You can just put a `not` in front of the `any` and take away the `== False`...

**Review by elehmer — CHANGES_REQUESTED** [review]
See my comments. The logic tree needs some work...if you want I can take a stab at it.

**Pull request merged**